presto-docs/src/main/sphinx/presto_cpp/properties.rst

```diff
@@ -277,3 +277,38 @@ The configuration properties of Presto C++ workers are described here, in alphab
 * **Default value:** ``0.25``
 
   See description for ``shared-arbitrator.memory-pool-min-free-capacity``
+
+LinuxMemoryChecker Properties
```

Comment:
The following memory properties don't have to be LinuxMemoryChecker specific?  Can we just call it Memory Pushback or Memory Checker properties?

Reply:
Changing it to be `Memory Checker Properties`.